README.md

```diff
@@ -367,6 +367,59 @@ Conversions are probably better done like this...
 Qty('0 tempC').add('100 degC') // => 100 tempC
 ```
 
+### Custom units and prefixes
+
+You can define your own custom units and prefixes by using the `Qty.defineUnit`
+function.
+
+```javascript
+Qty.defineUnit( unitName, unitDefinition[, isBase] ); 
+```
+
+`unitName` is the official name of the unit wrapped in `<` and `>`,
+e.g. `<meter>`
+
+`unitDefinition` is an array of definition values.  For units it is:
+ - `aliases` is an array of aliases for the unit, e.g. `["m","meter","meters","metre","metres"]`
+ - `conversion` is a decimal number that can be multiplied to the number to
+   convert to the base unit
+ - `kind` is the kind of the unit.  If you specify an unknown kind here, be
+   sure to include a unit with `isBase` set to true for doing conversions
+ - `numerator` is an array containing the set of units that can be specified as a numerator for this unit
+ - `denominator` is an array containing the set of units that can be specified as a denominator for this unit
```

Comment:
Why an array? Why not build the array yourself internally out of an object that's passed in? Less chance of accidentally swapping order that way.